sl/content.md

```diff
@@ -0,0 +1,89 @@
+# Scientific Linux
+
+%%LOGO%%
+
+This image containers for [Scientific Linux](http://www.scientificlinux.org/).
+
+The source for each release is in [its own branch](http://github.com/scientificlinux/sl-docker/branches) and contains the kickstart used to build the image.
+
+These images are maintained by the Scientific Linux Team in [github](https://github.com/scientificlinux/sl-docker).
```

Comment:
Since you have a shortened version of this in `maintainer.md`, this seems repetitive.  `maintainer.md` will end up in the Quick Reference section (ex. [arango](https://github.com/docker-library/docs/tree/3383325346b0ac532752d2f3725d1c654c3a3227/arangodb#quick-reference)) under the "Maintained By" bullet point.